src/components/Select/Select.md

```diff
@@ -1,9 +1,10 @@
 We leverage [React-Select](https://github.com/JedWatson/react-select) for our single and multi selects. For example, you can add
```

Comment:
is this what we expect our prettier config to do? Or is my setup messed up?

Reply:
It's a bit odd, we have `"singleQuote": true,` so I'd think it'd be single quotes